Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:edge
+
+WORKDIR /source
+ENV RABBITMQADMIN_VERSION="v3.9.0"
+RUN apk add --no-cache ca-certificates bash git openssh curl \
```

Comment:
@Mastermindzh would be better to use `alpine:3.16.2` instead of using `alpine:latest`, one of the rules that hadolint is introducing is this one:
1 DL3007 warning: Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

Reply:
![image](https://user-images.githubusercontent.com/111972490/187403205-ec82e981-d441-46a5-94c6-9479caaf86fd.png)
These are the remaining hadolint warnings and info.
Do you want to try add a specific version tag for each of the dependencies?

Reply:
I usually don't bother myself tbh :)

Reply:
We decided to go with the latest, it turned out to be was quite a job to get all the exact versions.